README.md

```diff
@@ -2,10 +2,6 @@
 
 This repository contains a Jenkins plugin implemented as a Maven project. The plugin provides functionality for performing Synopsys Security Scan with Black Duck, Coverity and Polaris. This README.md file serves as a guide for the users of the plugin. Please note that this Jenkins plugin currently supports only **Bitbucket as the source code management (SCM) system.**
 
-## Build
-
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-
```

Comment:
This shouldn't be removed

Reply:
@jahid1209 Can you elaborate on which feedback? Was this from Infra team?

Reply:
@dmamidisynopsys 
I meant that Tithi made some suggestion regarding sample pipeline in the README.md. Based on those suggestions **Maksud** made some changes. But he doesn't have the write permission to create a PR. Therefore, I created a PR on his behalf.